aws_marketplace/creating_marketplace_products/models/build-model-package-for-listing-in-aws-marketplace.ipynb

```diff
@@ -0,0 +1,1122 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.            "interval": 1 * SECOND,
</pre><p>Nit - add a comment stating how long this is, and rename SECOND to SECONDS.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3014/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>